inhibit/inhibit.go

```diff
@@ -117,11 +117,11 @@ func (ih *Inhibitor) Mutes(lset model.LabelSet) bool {
 
 	for _, r := range ih.rules {
 		if r.TargetMatchers.Match(lset) && r.hasEqual(lset) {
-			ih.marker.SetInhibited(fp, true)
+			ih.marker.SetStatus(fp, types.Inhibited)
 			return true
 		}
 	}
-	ih.marker.SetInhibited(fp, false)
+	ih.marker.SetStatus(fp, types.Active)
```

Comment:
If we don't know the real status until notify can re-check it again then maybe we should reset it into `types.Unprocessed`, wouldn't that make more sense?